### DISCUSSION TITLE
Fetch locator from QRZ

### DISCUSSION THREAD

**Discussion by n1zzo** [wavelog/wavelog#582] [Ideas]
When live logging a QSO, it would be nice to automatically fetch the locator of the correspondent from its QRZ Bio if available, most hams have that field populated, that would be a quality of life improvement!
I think the only gotcha is that you need to be logged in QRZ.com to access Bio fields.

**Comment by HB9HIL** [comment]
It already does that IF you have a XML Subscription by QRZ.com. 

Without Subscription no Locator
